src/app/modules/main/node_section/view.nim

```diff
@@ -0,0 +1,150 @@
+import NimQml, chronicles, json, strutils
+import io_interface
+import ../../../core/signals/types
+
+
+QtObject:
+  type
+    View* = ref object of QObject
+      delegate: io_interface.AccessInterface
+      callResult: string
+      lastMessage*: string
+      stats*: Stats
+      peerSize: int
+      bloomBitsSet: int
+ 
+  proc delete*(self: View) =
+    self.QObject.delete
+
+  proc newView*(delegate: io_interface.AccessInterface): View =
+    new(result, delete)
+    result.QObject.setup
+    result.delegate = delegate
+    result.callResult = "Use this tool to call JSONRPC methods"
+    result.lastMessage = ""
+
+  proc load*(self: View) =
+    self.delegate.viewDidLoad()
+
+  proc callResult*(self: View): string {.slot.} =
+    result = self.callResult
+
+  proc callResultChanged*(self: View, callResult: string) {.signal.}
+
+  proc setCallResult(self: View, callResult: string) {.slot.} =
+    if self.callResult == callResult: return
+    self.callResult = callResult
+    self.callResultChanged(callResult)
+
+  proc `callResult=`*(self: View, callResult: string) = self.setCallResult(callResult)
+
+  QtProperty[string] callResult:
+    read = callResult
+    write = setCallResult
+    notify = callResultChanged
+
+  proc onSend*(self: View, inputJSON: string) {.slot.} =
+    self.setCallResult(self.delegate.sendRPCMessageRaw(inputJSON))
+
+  proc onMessage*(self: View, message: string) {.slot.} =
+    self.setCallResult(message)
+
+  proc receivedMessage*(self: View, lastMessage: string) {.signal.}
+
+  proc setLastMessage*(self: View, lastMessage: string) {.slot.} =
+    self.lastMessage = lastMessage
+    self.receivedMessage(lastMessage)
+
+  QtProperty[string] lastMessage:
+    read = lastMessage
+    write = setLastMessage
+    notify = receivedMessage
+
+  proc getWakuBloomFilterMode*(self: View): bool {.slot.} =
+    result = self.delegate.getWakuBloomFilterMode()
+
+  proc getBloomLevel*(self: View): string {.slot.} =
+    var wakuBloomFilterMode = self.delegate.getWakuBloomFilterMode()
```

Comment:
@borismelnik this logic is not for the view, actually there is no any more `getWakuBloomFilterMode` method, seems this is not rebased for a while, now we have `getBloomLevel` containing all logic there, you just need to call it.